NoSQL/ce/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022 Oracle and/or its affiliates
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+#
+FROM openjdk:14-oraclelinux8
```

Comment:
The only OpenJDK in this repo is 17: https://github.com/oracle/docker-images/tree/main/OracleOpenJDK/17

You should be using that instead, if possible.